nemo/lightning/resume.py

```diff
@@ -169,6 +171,10 @@ def nemo_path(self, model=None) -> Optional[Path]:
         if checkpoint:
             if self.adapter_path:
                 return AdapterPath(checkpoint, adapter_path=Path(self.adapter_path))
+
+            model_weights_path = Path(checkpoint) / AutoResume.WEIGHTS_PATH
+            if os.path.isdir(model_weights_path):
```

Comment:
Why is it conditional? What's the case where we don't have the `weights` directory?

Reply:
I wanted to make `AutoResume` as flexible as possible and wanted to try to account for the case where the user is not using our `ModelCheckpoint`  class. But that codepath has not been tested and would likely have other bugs, so I can remove this conditional to be more explicit.